src/components/shared/MemberSearchBar.tsx

```diff
@@ -75,11 +83,19 @@ class MemberSearchBarView extends React.Component<
           onClearText={() => this.clearSuggestions()}
         />
         <FlatList
+          keyboardShouldPersistTaps="always"
```

Comment:
FYI this is for the issue listed here: https://github.com/facebook/react-native/issues/4229  -- the first tap on the item was dismissing the soft keyboard and the second tap would do the onPress. This property makes it so that the first tap goes to the onPress

Reply:
could you leave add this as a comment in code?